src/main/java/frc/robot/RobotContainer.java

```diff
@@ -114,10 +118,17 @@ public class RobotContainer {
   /** Constructor for {@link RobotContainer} */
   public RobotContainer() {
     configureBindings();
+    configureAuto();
 
     drive.setDefaultCommand(driveRobotOriented);
   }
 
+  private void configureAuto() {
+    autoChooser = new SendableChooser<>();
+    SmartDashboard.putData("choose auto", autoChooser);
+    /* This is where you put auto commands. */
```

Comment:
This isn't where you put auto commands. You'll need to add auto commands to the SendableChooser *before* you send it to SmartDash.